qiskit/providers/__init__.py

```diff
@@ -286,6 +299,28 @@ def _define(self):
 identify the gate in the basis set for the transpiler. If there is a conflict
 the transpiler will not know which gate to use.
 
+2. Add the custom gate to the target for your backend. This can be done with
+the :class:`~qiskit.transpiler.Target` class's
+:meth:`~qiskit.transpiler.Target.add_instruction` method. You'll need to
+add an instance of ``SYGate`` and it's parameters to the target so the transpiler
+knows it exists. For example, assuming this is part of your
+:class:~qiskit.providers.BackendV2` implementation for your backend::
```

Comment:
missing `` ` `` in the Sphinx directive

Reply:
Fixed in the current revision (just forced pushed as part of a rebase too)